modAionImpl/src/org/aion/zero/impl/blockchain/AionPendingStateImpl.java

```diff
@@ -1073,7 +1074,13 @@ private AionTxExecSummary executeTx(AionTransaction tx, boolean inPool) {
                         bestBlk.getNrgLimit(),
                         LOGGER_VM,
                         getPostExecutionWork());
-        return txExe.execute().get(0);
+        try {
+            return txExe.execute().get(0);
+        } catch (VMException e) {
+            LOGGER_VM.error(e.toString());
```

Comment:
similarly, needs the message and stack trace

Reply:
updated